src/lib/coda_commands/coda_commands.ml

```diff
@@ -429,7 +424,6 @@ let get_status ~flag t =
   ; consensus_time_now
   ; consensus_mechanism
   ; consensus_configuration
```

Comment:
why remove libp2p_peer_id from status? This has been useful in the past for users to know what their peerids are.

Reply:
It's folded into node_addrs_and_ports